infrastructure/builder/src/build/utils.js

```diff
@@ -326,7 +326,7 @@ exports.dockerRegistryCheck = async ({tag}) => {
     // Acces the registry API directly to see if this tag already exists, and do not push if so.
     // TODO: this won't work with custom registries!
     const res = await got(`https://index.docker.io/v1/repositories/${repo}/tags`, {json: true});
-    if (res.body && _.includes(res.body.map(l => l.name), imagetag)) {
+    if (res.body && res.body.map(l => l.name).includes(imagetag)) {
```

Comment:
What's `l`? Let's rename to something more descriptive.

Reply:
it's a common pattern to use non-descriptive one-letter names like this in lamda-functions. As to why `l` was chosen in this case - I am not sure :)